content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,306 @@
+---
+reviewers:
+- gjtempleton
+- jonathan-innis
+- maciekpytel
+title: Node Autoscaling
+linkTitle: Node Autoscaling
+description: >-
+  Automatically provision and consolidate the Nodes in your cluster to adapt to demand and optimize cost.
+content_type: concept
+weight: 130
```

Comment:
Now that we have https://kubernetes.io/docs/concepts/cluster-administration/node-shutdown/, I suggest
```suggestion
weight: 15
```